### DISCUSSION TITLE
如何使用流量开启代理

### DISCUSSION THREAD

**Discussion by bean0283** [1715173329/UnblockNeteaseMusic#955] [Q&A]
请问有没有办法使用流量的时候可以用，wifi下安卓可以正常使用，直接使用流量，使用第三方的http代理工具，无法使用，我尝试开热点给另外的手机，然后修改代理，但仍然无法使用，

**Comment by wukibaka** [comment]
试试修改apn的方式
![Screenshot_20221231_185344_com.android.settings_edit_330229274221484.jpg](https://user-images.githubusercontent.com/51240031/210134184-e1486d2d-6cb2-4bf7-879a-f8d3ef9e8185.jpg)
